src/TagBot/TagBot.jl

```diff
@@ -0,0 +1,117 @@
+module TagBot
+
+using Dates: Day, UTC, now
+
+using GitHub: GitHub
+using JSON: JSON
+
+const GH = GitHub
+
+const AUTH = Ref{GH.OAuth2}()
+const ISSUE_TITLE = "TagBot trigger issue"
+const ISSUE_BODY = """
+This issue is used to trigger TagBot; feel free to unsubscribe.
+
+If you haven't already, you should update your `TagBot.yml` to include issue comment triggers.
+Please see [this post on Discourse](TODO: Discourse URL) for instructions and more details.
+"""
+
+include("cron.jl")
+include("pull_request.jl")
+
+function main()
+    AUTH[] = GH.authenticate(ENV["GITHUB_TOKEN"])
```

Comment:
Is this correct? Shouldn't you be using your own PAT here, since you need to create an issue on a different repo? The `GITHUB_TOKEN` token won't be able to do so.

Reply:
Does this not work in a workflow file? 

```yml
env:
  GITHUB_TOKEN: ${{ secrets.SOME_OTHER_TOKEN }}
```

Reply:
I'm actually not sure. It might work. I thought GitHub prevented you from setting environment variables that begin with `GITHUB`, but I'm not sure.

Reply:
That kinda rings a bell, but [it does indeed work](https://github.com/christopher-dG/TestRepo/runs/1323931025?check_suite_focus=true#step:3:6).

Reply:
Sounds good